tools/tests/com.liferay.ide.xml.search.ui.tests/src/com/liferay/ide/xml/search/ui/tests/LiferayPortletXmlTests.java

```diff
@@ -96,12 +109,45 @@ public void testDDMDisplay()
     public void testFooterPortletCss()
     {
     }
+    
+    @Test
+    public void testFooterPortalCss() throws Exception
+    {
+    	if( shouldSkipBundleTests() ) return;
+    	
+        final IFile descriptorFile = getDescriptorFile();
+        final String elementName = "footer-portal-css";
+        final String elementValue = "footer-portal-css";
+  
+        setElementContent(descriptorFile, elementName, elementValue);
+        buildAndValidate( descriptorFile );
+        
+        String markerMessage=MessageFormat.format(LiferayBaseValidator.MESSAGE_RESOURCE_NOT_FOUND, new Object[] { elementValue });
+        assertFalse(checkMarkerByMessage(descriptorFile, MARKER_TYPE, markerMessage, true));
+    	
+    }
 
     // TODO
     public void testFooterPortletJavaScript()
```

Comment:
Can we add a test for this one at the same time?  Seems pretty straight forward.